project_structure.txt

```diff
@@ -0,0 +1,59 @@
+.modin/
+   # left `core` folder only since it mostly has changes
+   .core/
+      .dataframe/
+         .algebra/
+            ./d2p
```

Comment:
this should be changed to `default2pandas` as it was renamed in the repo in this PR

Reply:
Should be fixed

Reply:
I am planning on removing this file.